File: PHP/Human.php
<?php

    class Human{
        private $NIK = '';
        private $nama = '';
        private $jenis_kelamin = '';

        public function __construct($NIK = '', $nama = '', $jenis_kelamin = ''){
            $this->NIK = $NIK;
            $this->nama = $nama;
            $this->jenis_kelamin = $jenis_kelamin;
        }

        public function set_NIK($NIK){
            $this->NIK = $NIK;
        }
        public function get_NIK(){
            return $this->NIK;
        }

        public function set_nama($nama){
            $this->nama = $nama;
        }
        public function get_nama(){
            return $this->nama;
        }

        public function set_jenis_kelamin($jenis_kelamin){
            $this->jenis_kelamin = $jenis_kelamin;
        }
        public function get_jenis_kelamin(){
            return $this->jenis_kelamin;
        }
    }


?>

File: Python/Human.py
from Mahasiswa import Mahasiswa

class Human(Mahasiswa):

    __NIK = ""
    __nama = ""
    __jenis_kelamin = ""

    def __init__(self, NIK = "", nama = "", jenis_kelamin = ""):
        self.__NIK = NIK
        self.__nama = nama
        self.__jenis_kelamin = jenis_kelamin

    def get_NIK(self):
        return self.__NIK

    def set_NIK(self, NIK):
        self.__NIK = NIK

    def get_nama(self):
        return self.__nama

    def set_nama(self, nama):
        self.__nama = nama

    def get_jenis_kelamin(self):
        return self.__jenis_kelamin

    def set_jenis_kelamin(self, jenis_kelamin):
        self.__jenis_kelamin = jenis_kelamin